### PR TITLE
feat(ac-card-actions): Support Adaptive card actions

### PR DESCRIPTION
**Pull request by kishorepmr**
Description of changes made:

- Added onExecuteAction to every adaptive card to handle OpenUrl Action and Submit Action
- CardAction function in webex-js-sdk requires conversation url and parent activity id for which a card action has been performed, passed the conversations and parent activity id from Activity List

Associated issue number/tracking link:
94095

### CODE REVIEW

**Comment by adamweeks** [comment]
Please make sure the merge commits are not part of the PR. The PR should be rebased on the latest upstream master branch and only have commits with changes.

File: packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js [hunk 1]
```diff
@@ -49,13 +49,35 @@ export function getCardConfig(card, sparkInstance) {
...
     const PatchedAdaptiveCard = getPatchedAdaptiveCard(addChildNode);
     const adaptiveCard = new PatchedAdaptiveCard.AdaptiveCard();
 
+    adaptiveCard.onExecuteAction = (action) => {
+      switch (action.getJsonTypeName()) {
+        case OPEN_URL_ACTION:
+          window.open(action.url);
+          break;
+        case SUBMIT_ACTION:
+          sparkInstance.internal.conversation
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js]: Actions performed on the SDK should be done in the redux action and not within the utilities. This should be an action within the `redux-module-conversation` as well as a reducer state added to show the action being performed. Take a look at the redux styleguide for more info: https://github.com/webex/web-styleguide/tree/master/react/redux

**Comment by lalli-flores** [comment]
@kishorepmr Are you going to update this PR? It seems like it has fallen behind and there are conflicts to be resolved.
If not, please let me know so I can close this PR.

**Comment by mokandre** [comment]
> @kishorepmr Are you going to update this PR? It seems like it has fallen behind and there are conflicts to be resolved.
> If not, please let me know so I can close this PR.

@lalli-flores  I'm gonna work on these requested changes. So can close this PR and I'll do PR again when I'm done with the modifications

**Comment by lalli-flores** [comment]
Thanks @mokandre, I will close this PR

**Pull request closed without merging**